include/TMU.hpp

```diff
@@ -185,7 +187,79 @@ class TMU {
             .Data = CO_LINK(0x2100, 3, 16),
         },
 
-        /**
+        /*
+         * TPDO 1 Settings
+         * 0. The number of sub indices.
+         * 1. The COBID for the transmitting node
+         * 2. The transmission trigger 0xFE is asynchronous
+         * 3. The inhibit time
+         * 4. The event timer
+         */
+        {
+            .Key = CO_KEY(0x1801, 0, CO_UNSIGNED8 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 5,
+        },
+        {
+            //180h+Node-ID
+            .Key = CO_KEY(0x1801, 1, CO_UNSIGNED32 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) CO_COBID_TPDO_DEFAULT(0) + NODE_ID,
+        },
+        {
+            //timer triggered
+            .Key = CO_KEY(0x1801, 2, CO_UNSIGNED8 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 0xFE,
+        },
+        {
+            //no inhibit time
+            .Key = CO_KEY(0x1801, 3, CO_UNSIGNED16 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 0,
+        },
+        {
+            //send every 2 seconds
+            .Key = CO_KEY(0x1801, 5, CO_UNSIGNED16 | CO_OBJ_D__R_),
+            .Type = CO_TEVENT,
+            .Data = (uintptr_t) 500,
+        },
+
+        /*
+         * TPDO 1 Mapping
+         * 0. The number of mapping objects in the second TPDO
+         * 1. Link to the first error - err_arr[0]
+         * 2. Link to the second error - err_arr[1]
+         * 3. Link to the third error - err_arr[2]
+         * 4. Link to the fourth error - err_arr[3]
+         */
+        {
+            .Key = CO_KEY(0x1A01, 0, CO_UNSIGNED8 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 4,
+        },
+        {
+            .Key = CO_KEY(0x1A01, 1, CO_UNSIGNED32 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = CO_LINK(0x2101, 0, 8),
+        },
+        {
+            .Key = CO_KEY(0x1A01, 2, CO_UNSIGNED32 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = CO_LINK(0x2101, 1, 8),
+        },
+        {
+            .Key = CO_KEY(0x1A01, 3, CO_UNSIGNED32 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = CO_LINK(0x2101, 2, 8),
+        },
+        {
+            .Key = CO_KEY(0x1A01, 4, CO_UNSIGNED32 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = CO_LINK(0x2101, 3, 8),
+        },
+
+        /*
          * Data Links
          */
         {
```

Comment:
You need to add pointers to the error array here, so the data can be reported over CAN